### PR TITLE
fixed the grammatical errors in About Fossology page

### PR DESCRIPTION
**Pull request by Shruti3004**
## Description

Fixed the grammatical error in the documentation of about fossology.

### Changes

Updated the file src/www/ui/template/about.html.twig

## How to test
http://localhost/repo/?mod=about
Check the updated document on this.

Issue
#1935

### CODE REVIEW

**Comment by GMishx** [comment]
I am sorry, I did not notice this before but this page holds the license text and the Copyright information about FOSSology tool.

The changes done by you in the license text will not match the original license: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html and https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html .
I am not sure it is a good idea to edit them.

Also, some changes were done in the copyrights captured. That we can't do without the consent from the original authors.
That means the changes done in this PR can not be merged :-(

However, if you like, you can do a clearing on FOSSology and update the copyright list like https://github.com/fossology/fossology/blob/aaec8d420e0d58f4e7663a9694e1fae44b42def6/NOTICES#L926-L1469.
Just make sure to mark test data as irrelevant.

**Comment by Shruti3004** [comment]
> I am sorry, I did not notice this before but this page holds the license text and the Copyright information about FOSSology tool.
> 
> The changes done by you in the license text will not match the original license: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html and https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html .
> I am not sure it is a good idea to edit them.
> 
> Also, some changes were done in the copyrights captured. That we can't do without the consent from the original authors.
> That means the changes done in this PR can not be merged :-(
> 
> However, if you like, you can do a clearing on FOSSology and update the copyright list like
> 
> https://github.com/fossology/fossology/blob/aaec8d420e0d58f4e7663a9694e1fae44b42def6/NOTICES#L926-L1469
> 
> .
> Just make sure to mark test data as irrelevant.

Okay @GMishx

**Comment by Shruti3004** [comment]
@GMishx please review it now

**Comment by GMishx** [comment]
Unfortunately the PR still modifies the GPL-2.0 and LGPL-2.1 license text.

@mcjaeger , can you please review this?

File: src/www/ui/template/about.html.twig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {# Copyright 2014-2020 Siemens AG
 
    Copying and distribution of this file, with or without modification,
-   are permitted in any medium without royalty provided the copyright notice and this notice are preserved.
+   are permitted in any medium without royalty provided the copyright notice and this notice is preserved.
```
Comment (mcjaeger) [inline src/www/ui/template/about.html.twig:4]: no, let's not do this, change of license text

**Comment by mcjaeger** [comment]
Hello @Shruti3004 thank you so much for the contribution. but changing license text for grammatical issues is not worth the hazzle, even it is maybe not changing the licensing conditions, it might lead also to license scan issues (not resulting in 100% match which confuses people in aggregated views)

**Comment by mcjaeger** [comment]
I am very sorry, because I think we are thankful that you crafted a contribution to fossology, but it would be better maybe to close it (and not merge it), please reopen if you think I have overlooked and issue.

**Pull request closed without merging**